### ISSUE TITLE
Web Accessibility First Aid missing footer

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-website#13]
https://www.w3.org/WAI/beta/plan/interim-repairs/

**Comment by yatil** [comment]
This issue was moved to w3c/wai-first-aid#2

**Comment by yatil** [comment]
Moved to the appropriate repository.
